src/commands/project/use.js

```diff
@@ -0,0 +1,29 @@
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+
+class ProjectUse extends BaseCommand {
+  async run() {
+    await super.run();
+
+    const project = this.userConfig.getProjectById(this.args.project);
+    if (!project) {
+      this.logger.error('This project does not exist run twilio project:list to see a list of projects');
```

Comment:
This error could be a little better. How about _\`The project "${this.args.project}" does not exist. Run "twilio project:list" to see the list of configured projects.\`_